docs/features/building/helm-arguments.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Helm Arguments
+---
+
+## What are Helm Arguments?
+
+**Helm Arguments** are used to reconfigure the default configurations in **Helm Charts**. They are similar to environment variables. Each **branch** of a Custom Service KintoBlock will have their own Helm Arguments. It is also posisble to add multiple Helm Arguments to a Custom Service KintoBlock.
```

Comment:
"they are similar to environment variables" -> "They will be used during the deployment".
"Each **branch** of a Custom Service KintoBlock will have their own Helm Arguments." -> wrong, it's one for all.
"posisble" -> typo

Reply:
Done